webapp/sources/rudder/rudder-web/src/main/scala/bootstrap/liftweb/RudderConfig.scala

```diff
@@ -1187,10 +1189,9 @@ object RudderConfig extends Loggable {
    * lazy and will only be initialised on the first call to one
    * of its (public) methods.
    */
-  def init() : Unit = {
+  def init() : IO[SystemError, Unit] = {
 
-    // this one must be in a fork thread pool
-    ZioRuntime.internal.unsafeRunAsync(IOResult.effect {
```

Comment:
the "async" made exception disappear and not be catched, ever. Not sure what was  happening. I failed to understood why it needed to be more on an other threadpool than what is provided by `IOResult.effect`. It works on one cpu, with plugins with the patch, too.